scheduler/management/commands/trigger_deliver_tasks.py

```diff
@@ -0,0 +1,221 @@
+import os
+import re
+
+from django.utils.six.moves import input
+
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
+
+from demands import JSONServiceClient, HTTPServiceClient
+from django.core.management import BaseCommand, CommandError
+from django.core.validators import URLValidator
+from django.utils.dateparse import parse_datetime
+
+from scheduler.models import Schedule
+
+
+def get_schedule(schedule_string):
+    schedule_type, _, lookup_id = schedule_string.partition(':')
+    if schedule_type == 'crontab':
+        kwargs = {'celery_cron_definition': int(lookup_id)}
+    elif schedule_type == 'interval':
+        kwargs = {'celery_interval_definition': int(lookup_id)}
+    else:
+        raise CommandError(
+            'Invalid schedule format: %s, unknown type: %s' % (
+                schedule_string, schedule_type))
+    return Schedule.objects.filter(enabled=True).filter(**kwargs)
+
+
+def get_model(model_class):
+    def callback(pk):
+        try:
+            return model_class.objects.get(pk=int(pk))
+        except (model_class.DoesNotExist,) as e:
+            raise CommandError(e)
+        except (Exception,) as e:
+            raise CommandError(e)
+    return callback
+
+
+def mk_validator(validator_class):
+    def validator_callback(input_str):
+        validator = validator_class()
+        validator(input_str)
+        return input_str
+    return validator_callback
+
+
+class MessageSenderApiClient(object):
+    """
+    Client for Message Sender Service.
+    :param str auth_token:
+        An access token.
+    :param str api_url:
+        The full URL of the API.
+    """
```

Comment:
https://github.com/praekeltfoundation/seed-services-client/pull/15 is the PR for that, when that lands we can remove this code.